zebra-rpc/src/methods/get_block_template_rpcs.rs

```diff
@@ -299,7 +313,7 @@ where
                     data: None,
                 })?;
 
-            if estimated_distance_to_chain_tip > MAX_ESTIMATED_DISTANCE_TO_NETWORK_CHAIN_TIP {
+            if !sync_status.is_close_to_tip() || estimated_distance_to_chain_tip > MAX_ESTIMATED_DISTANCE_TO_NETWORK_CHAIN_TIP {
                 tracing::info!(
                     estimated_distance_to_chain_tip,
                     ?tip_height,
```

Comment:
Also, should the error number below be a constant, like:
https://github.com/ZcashFoundation/zebra/blob/d778caebb8d032d62ddf4bf9c9bd3fe3add93960/zebra-rpc/src/methods.rs#L46-L50

This stops us accidentally changing it and breaking clients.

Reply:
Great point, yes it should, fixed in [`6d256c0`](https://github.com/ZcashFoundation/zebra/pull/5769/commits/6d256c01103098bd2388d57e830ed9fb481c25d5).